schedule/schedule-appengine/src/main/i18n/locale-en.json

```diff
@@ -213,5 +215,17 @@
   "VALUE_MUST_BE_LESS_THAN": "Value must be less than {{value}}",
   "VIEW_SUBSCRIBERS": "View Subscribers",
   "WEDNESDAY_ABBREVIATED": "Wed",
-  "WIDGETS": "Widgets"
+  "WIDGETS": "Widgets",
+  "HISTORY_TYPE": "History event type",
+  "HT_ALL": "Show all events",
```

Comment:
Is the intention of adding the translations so that clients can eventually also see the events?